utils.sh

```diff
@@ -230,6 +234,10 @@ function node_map_to_install_config_hosts() {
       if [[ "$boot_mode" == "null" ]]; then
              boot_mode="UEFI"
       fi
+      static_interface=enp2s0
+      if [[ "$PROVISIONING_NETWORK_PROFILE" == "Disabled" ]]; then
+        static_interface=enp1s0
+      fi
```

Comment:
Maybe be cleaner if we created a variable in `common.sh` and assigned it based on PROVISIONING_NETWORK_PROFILE?

(That way this could still work if testing with baremetal servers)

Reply:
done.